### ISSUE TITLE
Curl Error while building docker container for tensorflow serving

### ISSUE THREAD

**Issue by chaithuzz2** [tensorflow/serving#445]
I am trying to follow the instructions on https://tensorflow.github.io/serving/docker.html 

I am running OSX El Capitan on a 4GB RAM macbook pro

But the process fails with the following log. Can anyone please tell me how to fix this issue ?

```
Sending build context to Docker daemon 1.387 GB
Step 1/13 : FROM ubuntu:14.04
14.04: Pulling from library/ubuntu
Digest: sha256:b2a55128abd84a99436157c2fc759cf0a525c273722460e6f8f9630747dfe7e8
Status: Image is up to date for ubuntu:14.04
 ---> 2ff3b426bbaa
Step 2/13 : MAINTAINER Jeremiah Harmsen <jeremiah@google.com>
 ---> Using cache
 ---> d1c9d5863fca
Step 3/13 : RUN apt-get update && apt-get install -y         build-essential         curl         git         libfreetype6-dev         libpng12-dev         libzmq3-dev         pkg-config         python-dev         python-numpy         python-pip         software-properties-common         swig         zip         zlib1g-dev         libcurl3-dev         &&     apt-get clean &&     rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 376ad9fa1840
Step 4/13 : RUN curl -fSsL -O https://bootstrap.pypa.io/get-pip.py &&     python get-pip.py &&     rm get-pip.py
 ---> Using cache
 ---> 827f54aebc08
Step 5/13 : RUN pip install enum34 futures mock six &&     pip install --pre 'protobuf>=3.0.0a3' &&     pip install -i https://testpypi.python.org/simple --pre grpcio
 ---> Using cache
 ---> dd7dcde9258a
Step 6/13 : RUN add-apt-repository -y ppa:openjdk-r/ppa &&     apt-get update &&     apt-get install -y openjdk-8-jdk openjdk-8-jre-headless &&     apt-get clean &&     rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 8e1049d6f88f
Step 7/13 : RUN echo "startup --batch" >>/root/.bazelrc
 ---> Using cache
 ---> f7c9a1ea5796
Step 8/13 : RUN echo "build --spawn_strategy=standalone --genrule_strategy=standalone"     >>/root/.bazelrc
 ---> Using cache
 ---> b2acf66e8d77
Step 9/13 : ENV BAZELRC /root/.bazelrc
 ---> Using cache
 ---> 8b4cc65e5fc8
Step 10/13 : ENV BAZEL_VERSION 0.4.2
 ---> Using cache
 ---> 2b5d6138f849
Step 11/13 : WORKDIR /
 ---> Using cache
 ---> 8004f053290c
Step 12/13 : RUN mkdir /bazel &&     cd /bazel &&     curl -fSsL -O https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh &&     curl -fSsL -o /bazel/LICENSE.txt https://raw.githubusercontent.com/bazelbuild/bazel/master/LICENSE.txt &&     chmod +x bazel-*.sh &&     ./bazel-$BAZEL_VERSION-installer-linux-x86_64.sh &&     cd / &&     rm -f /bazel/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
 ---> Running in fa6fbf0a8265
curl: (56) SSL read: error:00000000:lib(0):func(0):reason(0), errno 104
The command '/bin/sh -c mkdir /bazel &&     cd /bazel &&     curl -fSsL -O https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh &&     curl -fSsL -o /bazel/LICENSE.txt https://raw.githubusercontent.com/bazelbuild/bazel/master/LICENSE.txt &&     chmod +x bazel-*.sh &&     ./bazel-$BAZEL_VERSION-installer-linux-x86_64.sh &&     cd / &&     rm -f /bazel/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh' returned a non-zero code: 56

```

**Comment by chaithuzz2** [comment]
Removing <code>curl -fSsL -o /bazel/LICENSE.txt https://raw.githubusercontent.com/bazelbuild/bazel/master/LICENSE.txt && </code> from the build file helped

**Comment by StripedBanana** [comment]
I'm having the exact same problem. Tried removing the `curl` line with the `LICENSE.txt` you mentioned, but the same error popped up. I'm guessing the problem is in the first `curl` about the bazel installer script.

I tried monitoring traffic while this `curl` was being executed, and I found out it was downloading very slowly (~30 kB/s), and the file is around 200 MB. Maybe the command is timing out?
